package.json

```diff
@@ -32,5 +32,9 @@
     "lint-staged": "^16.2.1",
     "postcss": "^8.5.6",
     "tailwindcss": "^4.1.14"
+  },
+  "dependencies": {
+    "@nestjs/swagger": "^11.2.1",
```

Comment:
No need for adding dependencies here. 

As this is mono repo: 

There are like 3 different package.json files. 

So regarding the dependency segregation. 
Backend specific dependencies goes to backend/package.json - which you already have added
Same goes for Frontend. 
Only something which are common in nature to both of them will come under the root package.json like.